inc/components/post-featured-media/component.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Post featured media.
+ *
+ * Display the post featured media.
+ *
+ * @todo Update to remove all material UI.
+ *
+ * @package Irving_Components
+ */
+
+namespace WP_Irving;
+
+use WP_Irving\Component;
+
+if ( ! function_exists( '\WP_Irving\get_registry' ) ) {
+	return;
+}
+
+/**
+ * Register the component and callback.
+ */
+get_registry()->register_component_from_config(
+	__DIR__ . '/component',
+	[
+		'callback' => function( Component $component ): Component {
+
+			// Get the post ID from a context provider, or fallback to the global.
+			$post_id = $component->get_config( 'post_id' );
+			if ( 0 === $post_id ) {
+				$post_id = get_the_ID();
+			}
+
+			// Get and validate image url.
+			$image_url = get_the_post_thumbnail_url( $post_id );
+			if ( empty( $image_url ) ) {
+				return $component;
+			}
+
+			$wrapper = new Component( 'irving/container' );
+			$wrapper->set_config( 'max_width', 'lg' );
+			$wrapper->set_config( 'style', [ 'bottom-margin' => '1rem' ] );
+
+			$wrapper->append_child(
+				( new Component( 'material/card-content' ) )
```

Comment:
We shouldn't be using material components in our core library. This may need to be a @todo in another ticket.

Reply:
Created IRV-465 in addition to the @todo.